client/components/rewind-credentials-form/index.jsx

```diff
@@ -43,11 +43,13 @@ export class RewindCredentialsForm extends Component {
 		siteUrl: PropTypes.string,
 		labels: PropTypes.object,
 		requirePath: PropTypes.bool,
+		notices: PropTypes.bool,
```

Comment:
instead of notices can we update it to `hasNotices` ? this way the prop will be more clear.

Reply:
done in https://github.com/Automattic/wp-calypso/pull/40094/commits/7f711af2e0c7a17edd40ceed07639db6fd914bb8